migrations/20180510155218_add_hxl_tags.php

```diff
@@ -0,0 +1,253 @@
+<?php
+
+use Phinx\Migration\AbstractMigration;
+
+class AddHxlTags extends AbstractMigration
```

Comment:
I don't think this is worth taking the time to change, and it is predicated on existing code, but the complexity in loops of this approach to inserting data is somewhat over the top, hopefully, with the newer framework we can avoid having to do this in future. Sorry you had to write it!

While it is conceivable that someone could insert data between table creation and the insertion of the data, that's not a usecase we should support, I think. The table should be blanked before hand, locked and then the data inserted as a formatted db dump which would be more readable and significantly faster.

Legitimately, I am not saying to change this there are lots of earlier versions of this but some day down the line we should try to do some metricing of migration times so we can figure out costs and determine if we care more about efficiency or about approach.

I read this a few times, and it works - I believe - as expected.